.github/workflows/swift-toolchain.yml

```diff
@@ -881,17 +877,19 @@ jobs:
           show-progress: false
 
       - name: Install Python ${{ env.PYTHON_VERSION }} (Host)
+        if: matrix.arch == 'amd64' || inputs.build_arch == 'amd64'
```

Comment:
Why can we not do this on ARM64?

Reply:
I suppose the `actions/setup-python` action doesn't have the right python version for arm64. Hence the use of `nuget` in the arm64 version of this in the next step, as in https://github.com/compnerd/swift-build/pull/390

Reply:
Ah, okay. I think that because we are doing only native compilation, perhaps we can leave this as `matrix.arch == 'amd64'`?